### PR TITLE
Tidy up tailwind.config.js color names in preparation for an eventual dark mode 

### PR DESCRIPTION
**Pull request by SaySayo**
This PR adds a work-in-progress dark mode to Ocaml.org. cc @sabine

### CODE REVIEW

**Comment by sabine** [comment]
Thanks for checking that nothing breaks Sayo. Now it would be great to rebase on main. :+1:

**Comment by tmattio** [comment]
Thanks Sayo!

Are these coming from the design system? If yes, can we include a PDF export (or other) of the colour scheme from the design system?

**Comment by tmattio** [comment]
> If yes, can we include a PDF export (or other) of the colour scheme from the design system?

Or alternatively, move it to the public Figma file and add a link somewhere in the README/CONTRIBUTING guide

**Comment by sabine** [comment]
> Are these coming from the design system?

The color names we have here are consistent with the naming in the design system, but there's more work to do both with setting up colors in tailwind.config.js and in the design system on Figma. This is very much a work-in-progress on both sides: codebase and Figma converging towards the same names for the light mode and dark mode colors.

Putting a document with the Figma colors will be no trouble when the work in Figma has been completed.

ETA: Yes, referencing the Figma file would likely be even better.

File: src/ocamlorg_frontend/pages/packages.eml [hunk 1]
```diff
@@ -144,7 +144,7 @@ Layout.render
         <div class="flex flex-col text-center">
             <div class="text-primary-600 mb-4 font-bold">STABLE ECOSYSTEM</div>
             <h3 class="font-bold mb-8">Focus on Your Code, and Opam Takes Care of Distributing It.</h3>
```
Comment (christinerose) [inline src/ocamlorg_frontend/pages/packages.eml]:
```suggestion
            <h3 class="font-bold mb-8">Focus on Your Code, and opam Takes Care of Distributing It.</h3>
```

File: src/ocamlorg_frontend/pages/packages.eml [hunk 2]
```diff
@@ -173,7 +173,7 @@ Layout.render
                     <%s! Icons.shield_check "h-6 w-6 text-white" %>
                 </div>
                 <div class="text-lg font-semibold mb-4">Stability</div>
-                <div class="text-base text-body-400">The Opam project and package repository is maintained by a team of
+                <div class="text-base text-lighter">The Opam project and package repository is maintained by a team of
```
Comment (christinerose) [inline src/ocamlorg_frontend/pages/packages.eml]:
```suggestion
                <div class="text-base text-lighter">The opam project and package repository is maintained by a team of
```

**Review by christinerose — APPROVED** [review]
For my part it looks great! Just suggested the preferred capitalizations of opam. :)

**Pull request merged**